### PR TITLE
fix gen of docs for sdk

### PR DESCRIPTION
**Pull request by keertip**

### CODE REVIEW

**Comment by sethladd** [comment]
Should we generate the SDK docs in the CI system so we catch regressions like this?

File: lib/src/model.dart [hunk 1]
```diff
@@ -1258,7 +1258,17 @@ class Parameter extends ModelElement {
 
   @override
   String get _href {
-    throw new UnsupportedError('parameters do not have hrefs');
+    var p = _parameter.enclosingElement;
```
Comment (sethladd) [inline lib/src/model.dart]: Why does a parameter need _href ? Isn't _href only used when we're generating a page for it?

**Comment by sethladd** [comment]
We also need to add an HTML ID to all parameters on dedicated page.

File: lib/src/model.dart (hunk 1 above)
Comment (keertip) [inline lib/src/model.dart]: Also used to resolve references in comments - parameters are linked in comments for methods/functions.

**Pull request merged**